__tests__/app.test.js

```diff
@@ -289,3 +287,111 @@ describe("GET /api/articles/:article_id/comments --- error handling", () => {
 // });
 // });
 // });
+
+
+
+describe("PATCH /api/articles/:article_id", () => {
+  const validInput77 = { inc_votes: 77 };
+
+  test("200 status code received when calling api correctly", () => {
+    return request(app).patch("/api/articles/1").send(validInput77).expect(200);
+  });
+
+  test("data received in the format {article: [{}]} with a single object in the array when input body is valid", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send(validInput77)
+      .expect(200)
+      .then((data) => {
+        expect(data.body).toMatchObject({
+          article: expect.any(Object),
+        });
+        expect(Array.isArray(data.body.article)).toBe(true);
```

Comment:
I think I might be tempted to just send back the article object on a key of article rather than sending the article back inside an array, so the response body would look like `{ article: { title: "title", article_id: 12 ...etc } }`rather than `{ article: [ { title: "title", article_id: 12 ...etc } ] }`